scripts/docker-run-rootfs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
+
+ROOTFS_PATH="$(pwd)/bin/targets/x86/64/openwrt-x86-64-generic-rootfs.tar.gz"
+NETWORK_PREFIX="${NETWORK_PREFIX:-192.168.1}"
+NETWORK_NAME="openwrt-lan-$NETWORK_PREFIX"
+IMAGE_NAME="openwrt-rootfs:$NETWORK_PREFIX"
+
+[ -f "$ROOTFS_PATH" ] || echo "Couldn't find rootfs at $ROOTFS_PATH"
+
+cat <<EOT > "$(dirname $ROOTFS_PATH)/Dockerfile"
+FROM scratch
+ADD $(basename $ROOTFS_PATH) /
+RUN sed 's/pi_ip="192.168.1.1/pi_ip="$NETWORK_PREFIX.1"/'
+RUN sed 's/pi_broadcast="192.168.1.255/pi_broadcast="$NETWORK_PREFIX.255"/'
+RUN echo "console::askfirst:/usr/libexec/login.sh" >> /etc/inittab
+EXPOSE 22 80 443
```

Comment:
disabled networking by default, explicitly enable it by `--network`

Reply:
done